x-pack/filebeat/module/zeek/weird/ingest/pipeline.json

```
@@ -28,13 +28,15 @@
     {
       "set": {
         "field": "source.ip",
-        "value": "{{source.address}}"
+        "value": "{{source.address}}",
+        "if": "ctx.source.address != null"
```

I recommend using the null-safe operator. https://www.elastic.co/guide/en/elasticsearch/painless/master/painless-operators-reference.html#null-safe-operator

```suggestion
        "if": "ctx?.source?.address != null"
```